contracts/0.4.24/Lido.sol

```diff
@@ -475,10 +483,18 @@ contract Lido is StETHPermit, AragonApp {
             _clBalance
         );
 
+        uint256 rewardsBase = appearedValidators.mul(DEPOSIT_SIZE).add(preClBalance);
+        int256 clBalanceDiff = _clBalance > rewardsBase ?
```

Comment:
1) redundant use of safemath since we have an explicit comparison
2) can be simplified to: `int256 clBalanceDiff = int256(_clBalance) - int256(rewardsBase);`

Reply:
fixed